benchmarks/aeossp_standard/verifier/models.py

```
@@ -0,0 +1,286 @@
+    action_time_step_s: int
+    geometry_sample_step_s: int
+    resource_sample_step_s: int
+
+    @property
+    def horizon_seconds(self) -> int:
+        seconds = (self.horizon_end - self.horizon_start).total_seconds()
+        if seconds < 0:
+            raise ValueError("mission horizon_end must be after horizon_start")
+        if abs(seconds - round(seconds)) > NUMERICAL_EPS:
+            raise ValueError("mission horizon must be an integer number of seconds")
+        return int(round(seconds))
```

Ruff (0.15.9)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)